omnigibson/robots/manipulation_robot.py

```diff
@@ -413,11 +413,14 @@ def _add_arm_control_dict(self, fcns, arm):
         # -n_joints because there may be an additional 6 entries at the beginning of the array, if this robot does
         # not have a fixed base (i.e.: the 6DOF --> "floating" joint)
         # see self.get_relative_jacobian() for more info
+        # We also count backwards for the link frame because if the robot is fixed base, the jacobian returned has one
+        # less index than the number of links. This is presumably because the 1st link of a fixed base robot will
+        # always have a zero jacobian since it can't move. Counting backwards resolves this issue.
```

Comment:
wow is this the concrete bug you found?

Reply:
well this is another bug found, but i don't think this caused the major issue